Sources/SwiftyLLVM/AtomicRMWBinOp.swift

```diff
@@ -0,0 +1,112 @@
+internal import llvmc
+
+/// The type of an atomic read-modify-write binary operation.
+public enum AtomicRMWBinOp {
+
+  /// Set the new value and return the one old.
+  case xchg
+  /// Add a value and return the old one.
+  case add
+  /// Subtract a value and return the old one.
+  case sub
+  /// And a value and return the old one.
+  case and
+  /// Not-And a value and return the old one.
+  case nand
+  /// OR a value and return the old one.
+  case or
+  /// Xor a value and return the old one.
+  case xor
+  /// Sets the value if it's greater than the original using a signed comparison and return the old one.
+  case max
+  /// Sets the value if it's Smaller than the original using a signed comparison and return the old one.
+  case min
+  /// Sets the value if it's greater than the original using an unsigned comparison and return the old one.
+  case uMax
```

Comment:
Is is called `uMax` (with the capital `M`) in LLVM? I think in cases like this one we should not capitalize the different cases of an enum to remain as close as possible to LLVM.

Reply:
Yes. The LLVM name is `LLVMAtomicRMWBinOpUMax`.
I've made the first letter lower-case to be consistent with our code.

Reply:
LGTM then.